doc/vision.rst

```diff
@@ -0,0 +1,45 @@
+Emukit Vision
+==============
+
+.. contents::
+    :local:
+
+Preface about emulation
+________________________
+
+
+We see emulation comprising of three main parts:
+
+- **Models**. This is a probabilistic data-driven representation of the process/simulator that the user is working with. There is normally a modelling framework that is used to create a model. Examples: neural network, Gaussian process, random forest. 
+- **Methods**. Relatively low-level techniques that are aimed that either understanding, quantifying or using uncertainty that the model provides. Examples: Bayesian optimization, experimental design. 
+- **Tasks**. High level goals that owners of the process/simulator might be actually interested in. Examples: measure quality of a simulator, explain complex system behavior. 
+
+Typical workflow that we envision for a user interested in emulation is:
+
+1. Figure out which questions/tasks are important for them in regard to their process/simulation.
+2. Understand which emulation techniques are needed to accomplish the chosen task.
+3. Build an emulator of the process. That can be a very involved step, that may include a lot of fine tuning and validation.
+4. Feed the emulator to the chosen technique and use it to answer the question/complete the task. 
+
+Emukit and emulation
+_____________________
+
+Here is Emukit approach towards each of three parts of emulation.
+
+Methods
+^^^^^^^^
+
+This is the main focus of Emukit. Emukit defines a general sctructure of a decision making method, called ``OuterLoop``, and then offers implementations of few such methods: Bayesian optimization, sensitivity analysis, experimental design. All methods in Emukit are model-agnostic.
```

Comment:
Let's remove sensitivity analysis from this paragraph and add a sentence like this: 'In addition to provide a framework for decision making Emukit provide other tools, like sensitivity analysis, that help to debug and interpret emulators'.

Reply:
quadrature is also not model agnostic.

Reply:
@mmahsereci , this is in "modelling framework" sense